docs/user/faq.rst

```diff
@@ -1,65 +1,71 @@
+  Formulate them as a question and an answer.
+  Consider that the answer is best as a reference to another place in the documentation.
 
-If you are still seeing errors because of C library dependencies,
-please see :ref:`faq:I get import errors on libraries that depend on C modules`.
 
+Building and publishing your project
+------------------------------------
 
-Help, my build passed but my documentation page is 404 Not Found!
------------------------------------------------------------------
 
-This often happens because you don't have an `index.html` file being generated.
-Make sure you have one of the following files:
+.. Old reference
+.. _My project isn't building correctly:
 
-    * `index.rst`
-    * `index.md`
+Why does my project have status "failing"?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-At the top level of your built documentation,
-otherwise we aren't able to serve a "default" index page.
+Projects have the status "failing" because something in the build process has failed.
+This can be because the project is not correctly configured,
+because the contents of the Git repository cannot be built,
+or in the most rare cases because a system that Read the Docs connects to is not working.
 
-To test if your docs actually built correctly,
-you can navigate to a specific page (`/en/latest/README.html` for example).
+First, you should check out the Builds tab of your project.
+By clicking on the failing step,
+you will be able to see details that can lead to resolutions to your build error.
 
+If the solution is not self-evident,
+you can use an important word or message from the error to search for a solution.
 
-My documentation requires additional dependencies
--------------------------------------------------
+.. seealso::
 
-For most Python dependencies, you can can specify a requirements file
-which details your dependencies. See our guide on :ref:`guides/reproducible-builds:using a configuration file`.
-You can also set your project documentation to install your project itself
-as a dependency.
+   :doc:`/build-troubleshooting`
+      Common errors and solutions for build failures.
 
-Your build may depend on extensions that require additional system packages to
-be installed. If you are using a :doc:`Configuration File <config-file/v2>` you
-can :ref:`add libraries with apt <config-file/v2:build.apt_packages>` to the
-Ubuntu-based builder .
+   Other FAQ entries
+      * :ref:`faq:My documentation requires additional dependencies`
+      * :ref:`faq:I get import errors on libraries that depend on C modules`
 
-If your project or its dependencies rely on C libraries that cannot be installed
-this way, see :ref:`faq:I get import errors on libraries that depend on C modules`.
 
+.. Old reference
+.. _Help, my build passed but my documentation page is 404 Not Found!:
 
-My project requires some additional settings
---------------------------------------------
+My build says "passed" but I get a 404 page - why?
```

Comment:
```suggestion
Why my build says "passed" but I get a 404 page?
```

Isn't it better/simpler just making it as a question?

Reply:
> That's not an entire sentence

What do you mean by that?

Reply:
It's not grammatically correct. You'd need something like

`Why does my build says "passed" but I get a 404 page?`

I agree that it's a better phrasing than the awkward `why` at the end tho.

Reply:
Ah, it was missing the "does". OK. Makes sense. Thanks for the explanation.

Yeah, I also prefer this "Why does..." form than the "- why?" one 👍🏼

Reply:
> Why does my build say~s~ "passed" but I get a 404 page?

Reply:
https://github.com/readthedocs/readthedocs.org/pull/9919